src/modules/rlm_smtp/all.mk.in

```diff
@@ -1,12 +1,16 @@
+ifneq "@targetname@" ""
```

Comment:
It'd be better to provide the curl version in the libfreeradius-curl all.mk file and then check that in the all.mk files that include it

Reply:
https://github.com/FreeRADIUS/freeradius-server/pull/3785